### PR TITLE
Update sso docs

### PR DESCRIPTION
**Pull request by OmriGez**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/sso-rbac/sso-providers/SAML/SAML.md
[the remark this answers is not in the document]
Comment (MPTG94) [reply docs/sso-rbac/sso-providers/SAML/SAML.md:1]: Please also update the SAML folder to saml

File: docs/sso-rbac/sso-providers/SAML/SAML.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+import DocCardList from '@theme/DocCardList';
+
+This documentation will describe the process of integrating a SAML application with Port, along with some examples for specific login providers.
+
+In order to integrate Port with a SAML SSO application, you will need to do the following:
+
+1. Create a new SAML application in your login provider.
+2. Share with us the following information about your SSO application: X509 certificate (`.pem` or `.crt` file) and Signin URL (as defined in the application).
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/SAML.md:9]: Did you end up testing with a `.crt` certificate to verify that it also works?

File: docs/sso-rbac/sso-providers/SAML/SAML.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+3. Port will provide you with a metadata XML file that you can upload in the SSO application to complete the connection.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/SAML.md:10]:
```suggestion
3. Port will provide you with a metadata XML file that you can upload in the identity provider interface, to the SSO application to complete the connection.
```

File: docs/sso-rbac/sso-providers/SAML/SAML.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+4. Update the application with the following attributes (Port expects the IdP to send the following attributes and their values in the authentication request):
+    * User attribute mappings:
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/SAML.md:12]: Change these `*` to `-`

Comment (MPTG94) [reply docs/sso-rbac/sso-providers/SAML/SAML.md:12]: across the whole doc

Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/SAML.md:12]: Add here in brackets what is the source and what is the target. For example - `given_name` is the target field the mapping will send to, `firstname` is the source field in the IdP that the value will be coming from

File: docs/sso-rbac/sso-providers/SAML/SAML.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        * email : email
+        * given_name : firstname
+        * family_name : lastname
+    * Constant attributes:
+        * email_verified : true
+
+If your login provider does not support metadata files, extract the following data from the XML and manually add to your application:
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/SAML.md:19]: This is not true, if they do not support XML upload, we will directly send them the correct values for the fields (or all they will have to do is input the `{CONNECTION_NAME}` parameter based on what we give them)

File: docs/sso-rbac/sso-providers/SAML/SAML.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+* IdP EntityID = `https://auth.getport.io`
+* SP EntityID = The EntityDescriptor field in the XML, looks like : `urn:auth0:port-prod:...` where the `...` is the connection name as provided to you by Port.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/SAML.md:22]:
```suggestion
* SP EntityID = The EntityDescriptor field in the XML, looks like : `urn:auth0:port-prod:{CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
```

File: docs/sso-rbac/sso-providers/SAML/SAML.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+* callback URL = The AssertionConsumerService binding field in the XML, looks like : `https://auth.getport.io/login/callback?connection=...` where the `...` is the connection name as provided to you by Port.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/SAML.md:23]:
```suggestion
* callback URL = The AssertionConsumerService binding field in the XML, looks like : `https://auth.getport.io/login/callback?connection={CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
```

File: docs/sso-rbac/sso-providers/SAML/google-workspace.md [hunk 7]
```diff
@@ -5,17 +5,17 @@ description: Integrate Google Workspace with Port
 ---
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/google-workspace.md:5]: Update the `sidebar_position` field

File: docs/sso-rbac/sso-providers/SAML/jumpcloud.md [hunk 8]
```diff
@@ -25,11 +25,11 @@ In order to complete the process you will need to contact Port to deliver and re
 1. In the Admin Portal, go to User Authentication -> SSO.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/jumpcloud.md:25]: Update the `sidebar_position` field

File: docs/sso-rbac/sso-providers/SAML/azure-ad.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+## Register a new application and generate the required credentials
+
+1. Head to your Azure portal and enter the Entra ID portal.
+2. Click on `Enterprise Applications` and create a new application.
+3. Click on `Create your own application` and select "Integrate any other application you don't find in the gallery (Non-gallery)".
+4. In the getting started section, select the "Set up Single Sign On" button, and choose the SAML method.
+5. Under `SAML certificates` (step 3), Click the Edit button to expand the certificates section. Click the 3 dot icon next to the Active certificate, and choose `PEM certificate download`.
+6. 
+
+Send the PEM certificate file along with the application ID (Which can be found in the properties tab of the application) to Port.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/azure-ad.md:31]: Should this be step number 6? or was there something else that should have been there?

File: docs/sso-rbac/sso-providers/SAML/azure-ad.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+After that, Port will provide you with a metadata XML file for your connection.
+
+At the top of the SAML configuration page, you can find a `upload metadata file` button. Press that one and upload the file provided to you by Port to complete the connection configuration.
+
+## Expose groups
+
+In order to expose your Azure groups to Port via the application, do the following:
+1. Head to the `Single sign on` section in the application configuration, and edit the `Attributes & Claims`.
+2. Press the `Add a group claim` button
+3. Select `Groups assigned to the application` and in the source attribute selectr `Cloud-only group display names`.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/azure-ad.md:42]:
```suggestion
3. Select `Groups assigned to the application` and in the source attribute select `Cloud-only group display names`.
```

File: docs/sso-rbac/sso-providers/SAML/azure-ad.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+4. Assign the relevant groups you want to expose to the application.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/SAML/azure-ad.md:43]:
```suggestion
4. Assign the relevant groups you want to expose to the application, these will be ingested into Port as teams you can use to manage user permissions and RBAC in your Port account.
```

**Review by MPTG94 — CHANGES_REQUESTED** [review]
Apply changes based on my comments

File: docs/sso-rbac/sso-providers/SAML/azure-ad.md (hunk 9 above)
Comment (OmriGez) [reply docs/sso-rbac/sso-providers/SAML/azure-ad.md:31]: typo, fixed

File: docs/sso-rbac/sso-providers/saml/saml.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+In order to integrate Port with a SAML SSO application, you will need to do the following:
+
+1. Create a new SAML application in your identity provider.
+2. Share with us the following information about your SSO application: X509 certificate (`.pem` file) and Signin URL (as defined in the application).
+3. Port will provide you with a metadata XML file that you can upload in the identity provider interface, to the SSO application to complete the connection.
+4. Update the application with the following attributes (Port expects the IdP to send the following attributes and their values in the authentication request, for example: `firstname` is the attribute in the IdP provider, and `given_name` is the key Port expects to receive that data in the authorization mapping):
+    - User attribute mappings:
+        - email : email
+        - given_name : firstname
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/saml.md:21]:
```suggestion
        - `given_name`: `firstname`
```

File: docs/sso-rbac/sso-providers/saml/saml.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+        - family_name : lastname
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/saml.md:22]:
```suggestion
        - `family_name`: `lastname`
```

File: docs/sso-rbac/sso-providers/saml/saml.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+    - Constant attributes:
+        - email_verified : true
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/saml.md:24]:
```suggestion
        - `email_verified`: `true`
```

File: docs/sso-rbac/sso-providers/saml/saml.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+If your login provider does not support metadata files, use the following information:
+
+* IdP EntityID = `https://auth.getport.io`
+* SP EntityID = The EntityDescriptor field in the XML, looks like : `urn:auth0:port-prod:{CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
+* callback URL = The AssertionConsumerService binding field in the XML, looks like : `https://auth.getport.io/login/callback?connection={CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/saml.md:30]:
```suggestion
* callback URL = The AssertionConsumerService binding field in the XML, looks like: `https://auth.getport.io/login/callback?connection={CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
```

File: docs/sso-rbac/sso-providers/saml/jumpcloud.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "JumpCloud"
-sidebar_position: 4
+sidebar_position: 3
 description: Integrate JumpCloud with Port
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/jumpcloud.md:4]:
```suggestion
description: Integrate JumpCloud with Port using SAML
```

File: docs/sso-rbac/sso-providers/saml/google-workspace.md [hunk 17]
```diff
@@ -1,21 +1,21 @@
 ---
 title: "Google Workspace"
-sidebar_position: 5
+sidebar_position: 2
 description: Integrate Google Workspace with Port
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/google-workspace.md:4]:
```suggestion
description: Integrate Google Workspace with Port using SAML
```

File: docs/sso-rbac/sso-providers/saml/azure-ad.md [hunk 18]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Azure-AD"
+sidebar_position: 1
+description: Integrate Azure AD with Port using SAML application
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/azure-ad.md:4]:
```suggestion
description: Integrate Azure AD with Port using SAML
```

File: docs/sso-rbac/sso-providers/saml/saml.md [hunk 19]
```diff
@@ -0,0 +1,32 @@
...
+
+This documentation will describe the process of integrating a SAML application with Port, along with some examples for specific identity providers.
+
+In order to integrate Port with a SAML SSO application, you will need to do the following:
+
+1. Create a new SAML application in your identity provider.
+2. Share with us the following information about your SSO application: X509 certificate (`.pem` file) and Signin URL (as defined in the application).
+3. Port will provide you with a metadata XML file that you can upload in the identity provider interface, to the SSO application to complete the connection.
+4. Update the application with the following attributes (Port expects the IdP to send the following attributes and their values in the authentication request, for example: `firstname` is the attribute in the IdP provider, and `given_name` is the key Port expects to receive that data in the authorization mapping):
+    - User attribute mappings:
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/saml.md:19]:
```suggestion
    - User attribute mappings - `target attribute name (Port)`: `source attribute name (SSO app)`:
```

File: docs/sso-rbac/sso-providers/saml/azure-ad.md [hunk 20]
```diff
@@ -0,0 +1,42 @@
...
+
+- Connect to the Port application via an AzureAD app.
+- Your AzureAD teams will be automatically synced with Port upon a user sign-in.
+- Set granular permissions on Port according to your AzureAD groups.
+
+## Register a new application and generate the required credentials
+
+1. Head to your Azure portal and enter the Entra ID portal.
+2. Click on `Enterprise Applications` and create a new application.
+3. Click on `Create your own application` and select "Integrate any other application you don't find in the gallery (Non-gallery)".
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/azure-ad.md:26]:
```suggestion
3. Click on `Create your own application` and select `Integrate any other application you don't find in the gallery (Non-gallery)`.
```

File: docs/sso-rbac/sso-providers/saml/azure-ad.md [hunk 21]
```diff
@@ -0,0 +1,42 @@
...
+4. In the getting started section, select the "Set up Single Sign On" button, and choose the SAML method.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/azure-ad.md:27]:
```suggestion
4. In the getting started section, select the `Set up Single Sign On` button, and choose the SAML method.
```

File: docs/sso-rbac/sso-providers/saml/azure-ad.md [hunk 22]
```diff
@@ -0,0 +1,42 @@
...
+5. Under `SAML certificates` (step 3), Click the Edit button to expand the certificates section. Click the 3 dot icon next to the Active certificate, and choose `PEM certificate download`.
+
+Send the PEM certificate file along with the application ID (Which can be found in the properties tab of the application) to Port.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/azure-ad.md:30]:
```suggestion
Send the **PEM certificate file** along with the **application ID** (Which can be found in the properties tab of the application) to Port.
```

File: docs/sso-rbac/sso-providers/saml/azure-ad.md [hunk 23]
```diff
@@ -0,0 +1,42 @@
...
+
+After that, Port will provide you with a metadata XML file for your connection.
+
+At the top of the SAML configuration page, you can find a `upload metadata file` button. Press that one and upload the file provided to you by Port to complete the connection configuration.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/azure-ad.md:34]:
```suggestion
At the top of the SAML configuration page, click on the `upload metadata file` button and upload the file provided to you by Port to complete the connection configuration.
```

File: docs/sso-rbac/sso-providers/saml/saml.md [hunk 24]
```diff
@@ -0,0 +1,32 @@
...
+    - User attribute mappings - `target attribute name (Port)`: `source attribute name (SSO app)`:
+        - `email`: `email`
+        - `given_name`: `firstname`
+        - `family_name`: `lastname`
+    - Constant attributes:
+        - `email_verified`: `true`
+
+If your login provider does not support metadata files, use the following information:
+
+* IdP EntityID = `https://auth.getport.io`
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/saml.md:28]:
```suggestion
- IdP EntityID: `https://auth.getport.io`
```

File: docs/sso-rbac/sso-providers/saml/saml.md [hunk 25]
```diff
@@ -0,0 +1,32 @@
...
+* SP EntityID = The EntityDescriptor field in the XML, looks like : `urn:auth0:port-prod:{CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/saml.md:29]:
```suggestion
- SP EntityID: The EntityDescriptor field in the XML, looks like : `urn:auth0:port-prod:{CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
```

File: docs/sso-rbac/sso-providers/saml/saml.md [hunk 26]
```diff
@@ -0,0 +1,32 @@
...
+* callback URL = The AssertionConsumerService binding field in the XML, looks like: `https://auth.getport.io/login/callback?connection={CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/saml.md:30]:
```suggestion
- callback URL: The AssertionConsumerService binding field in the XML, looks like: `https://auth.getport.io/login/callback?connection={CONNECTION_NAME}` where the `{CONNECTION_NAME}` is the connection name as provided to you by Port.
```

**Review by MPTG94 — APPROVED** [review]
Final changes

File: docs/sso-rbac/sso-providers/saml/azure-ad.md [hunk 27]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Azure-AD"
```
Comment (MPTG94) [inline docs/sso-rbac/sso-providers/saml/azure-ad.md:2]:
```suggestion
title: "Azure Active Directory (SAML)"
```

**Review by MPTG94 — APPROVED** [review]
Small update

**Pull request merged**